src/txs/Tx.tsx

```diff
@@ -464,12 +464,12 @@ function Tx<TxValues>(props: Props<TxValues>) {
   const modal = !error
     ? undefined
     : {
-        title: error?.toString().includes("UserDenied")
+        title: error?.toString().includes("User denied")
           ? t("Transaction was denied by user")
           : t("Error"),
-        children: error?.toString().includes("UserDenied") ? null : (
+        children: error?.toString().includes("User denied") ? null : (
           <Pre height={120} normal break>
-            {error?.message}
+            {error?.message ?? error?.toString()}
```

Comment:
Due to the fact that error messages are normally in `json` format, it would be better to use `JSON.stringify` here.

As is:

<img width="604" alt="Screenshot 2023-09-03 at 11 21 06 PM" src="https://github.com/terra-money/station/assets/30275692/b034c48e-128b-4ba7-84ed-c62d433e29d0">

With `JSON.stringify`:

<img width="608" alt="Screenshot 2023-09-03 at 11 21 50 PM" src="https://github.com/terra-money/station/assets/30275692/dc0a67ca-f314-4c85-8b6d-8641e4ee811f">

Reply:
I improved the parsing and added a user-readable message for some of the most common Ledger errors.